sim/app/api/webhooks/test/route.ts

```
@@ -224,6 +224,55 @@ export async function GET(request: NextRequest) {
         })
       }
 
+      case 'slack': {
+        const signingSecret = providerConfig.signingSecret || ''
+        
+        if (!signingSecret) {
```

This condition will never be true since in the previous line, we are defaulting `signingSecret` to `''`. I think the best path forward is to remove the default value, that way if the signingSecret is not actually provided then we can throw this error.

---

Yes you are right. I have remove the || condition, and since we are defaulting the providerCOnfig with an empty `{}` so in case signingSecret is missing it will return undefined and the condition will be true.